worldedit_commands/init.lua

```diff
@@ -516,9 +516,25 @@ minetest.register_chatcommand("/cylinder", {
 	end, check_cylinder),
 })
 
-minetest.register_chatcommand("/pyramid", {
+local check_pyramid = function(name, param)

```

Comment:
Should yes, it would be clearer. But I've followed the "convention" used in the rest of the file for check_replace, check_sphere, check_dome and check_cylinder.
So I would either stay like this or change all of them. What do you prefer ?